### DISCUSSION TITLE
Can you add vim-indent-object support ?

### DISCUSSION THREAD

**Discussion by momadacoding** [JetBrains/ideavim#318] [General]
It very useful for python programmer.
If this feature could not be added,  can you gays share some tips for working in python file ?

ref this https://youtrack.jetbrains.com/issue/VIM-1151

**Comment by AlexPl292** [comment]
Hi, this feature is on the way. Will be shipped with the next release.
https://github.com/JetBrains/ideavim/pull/261

**Reply by momadacoding** [reply]
It has bugs, I just leave a comment on that https://github.com/JetBrains/ideavim/pull/261#issuecomment-857316632
